apps/namadillo/src/App/Staking/ReDelegate.tsx

```diff
@@ -173,14 +173,16 @@ export const ReDelegate = (): JSX.Element => {
                 })
               )}
               extraContent={
-                <>
-                  <Alert
-                    type="warning"
-                    className="absolute py-3 right-3 top-4 max-w-[50%] text-xs rounded-sm"
-                  >
-                    To proceed all re-delegated value must be assigned
-                  </Alert>
-                </>
+                step === "assign" && (
+                  <>
+                    <Alert
+                      type="warning"
+                      className="absolute py-3 right-3 top-4 max-w-[50%] text-xs rounded-sm"
+                    >
+                      To proceed all re-delegated value must be assigned
```

Comment:
Super nitpicky, but adding a comma would make this read more nicely: `To proceed, all re-delegated value must be assigned`